src/index.tsx

```diff
@@ -1,15 +1,3 @@
-import React from 'react';
-import { makeDecorator } from '@storybook/addons';
-import { MockedProvider } from '@apollo/client/testing';
-
-export const withApolloClient = makeDecorator({
-  name: 'ApolloClient',
-  parameterName: 'apolloClient',
-  wrapper(getStory, context, settings) {
-    return (
-      <MockedProvider {...settings.options} {...settings.parameters}>
-        {getStory(context)}
-      </MockedProvider>
-    );
-  },
-});
+if (module && module.hot && module.hot.decline) {
+  module.hot.decline();
```

Comment:
@shilman not sure if I should add this in other places, I saw it in a few other addons.

Reply:
it's a minor optimization. i don't know how important it is.